### DISCUSSION TITLE
Notes not always updated when exporting bibliography as Better BibTex

### DISCUSSION THREAD

**Discussion by matthieu-str** [retorquere/zotero-better-bibtex#3061] [Q&A] [answered]
Hi all, 

I got the following problem: I regularly update notes and then export a shared library as a Better BibTex file, but for some reason, the notes I just wrote into are not updated in the resulting bib file. It might then take several days until everything is well updated. Does anyone know what's going on and how to fix this?

**Comment by retorquere** [comment, marked as answer]
sounds like a bug, please open an issue.
